components/forms/Multiselect/Multiselect.styles.ts

```diff
@@ -66,7 +75,9 @@ export const ListItem = styled.li<ListItemProps>`
   padding-block-end: ${basePadding(3)};
   ${tTruncate}
 
-  ${({ highlighted }) => highlighted && `background-color: var(--brand20)`}
```

Comment:
Same here - if you hovered over a list item, the list would flash again. Switching to css attribute selector fixed the issue.